test/resources/valid/ValidLargeProgramWithImports.flo

```diff
@@ -0,0 +1,50 @@
+import './UrlsImport' as Urls;
+import './UpdateUrlsImport' as UpdateUrls;
+
+action GET_ALL_POSTS = network(url = Urls.POSTS_URL, method = 'GET');
+action DELETE_ALL_POSTS = network(url = Urls.POSTS_URL, method = 'DELETE');
+
+action GET_ALL_USERS = network(url = Urls.USERS_URL, method = 'GET');
+action UPDATE_USER = network(url = UpdateUrls.UPDATE_USER_URL, method = 'PUT');
+
+action INCREMENT_COUNTER = counter();
+
+action TOGGLE_LOGIN_MODAL_OPEN = toggle();
+action TOGGLE_SIGNUP_MODAL_OPEN = toggle();
+
+action TOGGLE_SIGNED_IN = toggle();
+
+action STUB_ACTION = stub();
+
+flow POSTS = {
+    array posts = [];
+    GET_ALL_POSTS >> posts;
+    DELETE_ALL_POSTS >> posts;
+}
+
+flow USERS = {
+    array users = [];
+    GET_ALL_USERS >> users;
+    UPDATE_USER >> users;
+}
+
+flow COUNTER = {
+    number count = 0;
+    INCREMENT_COUNTER >> count;
+}
+
+flow MODALS = {
+    boolean isLoginModalOpen = false;
+    boolean isSignupModalOpen = false;
+    TOGGLE_LOGIN_MODAL_OPEN >> isLoginModalOpen;
+    TOGGLE_SIGNUP_MODAL_OPEN >> isSignupModalOpen;
+}
+
+flow ME = {
+    boolean isSignedIn = false;
+    TOGGLE_SIGNED_IN >> isSignedIn;
+}
+
+flow MY_FUN_STUB_FLOW = {
+    STUB_ACTION;
```

Comment:
What does it mean when an action is used in a flow and does not modify any fields? (i.e. What would be codegen'ed here @JeffreyDoyle?)

Reply:
I stole this example from @JeffreyDoyle so I thought this as a use case